x-pack/auditbeat/module/system/system.go

```diff
@@ -10,13 +10,9 @@ import (
 	"github.com/elastic/go-sysinfo"
 )
 
-const (
-	moduleName = "system"
-)
-
 func init() {
 	// Register the custom ModuleFactory function for the system module.
-	if err := mb.Registry.AddModule(moduleName, NewModule); err != nil {
+	if err := mb.Registry.AddModule(ModuleName, NewModule); err != nil {
```

Comment:
I think this will affect `event.module` and possibly `event.dataset` field values (assuming this string goes into those fields). Within Fleet integrations these are `constant_keyword` fields with a static `value`. If the event document differs from the mapping then it will fail to index. So if this is true then we need to make more adjustments so that the field value does not change under agentbeat.

https://github.com/elastic/integrations/blob/77cf4288b00d76b0838f7e4275a297bcd37877c8/packages/auditd_manager/data_stream/auditd/fields/base-fields.yml#L10-L13

Reply:
I have fixed this in the PR, it no longer changes the name of the module.